core/src/consensus/tendermint/worker.rs

```diff
@@ -683,9 +683,9 @@ impl Worker {
                 } else {
                     let parent_block_hash = self.prev_block_hash();
                     if self.is_signer_proposer(&parent_block_hash) {
-                        if let TwoThirdsMajority::Lock(lock_view, _) = self.last_two_thirds_majority {
+                        if let TwoThirdsMajority::Lock(lock_view, locked_block_hash) = &self.last_two_thirds_majority {
```

Comment:
`TwoThirdsMajority` derives `Copy`. Would you check again it requires to take a reference?